src/gui_common/GUICommon.cs

```diff
@@ -111,6 +111,32 @@ public static void MarkInputAsValid(LineEdit control)
         control.Set("custom_colors/font_color", new Color(1, 1, 1));
     }
 
+    /// <summary>
+    ///   Makes a string safe to store as a meta tag in rich text.
+    /// </summary>
+    public static string ConvertToSafeMetaTag(string tagContent)
```

Comment:
I think this and the next method would make more sense to be in `StringUtils` class than here.